identity/src/main/java/com/android/identity/credential/CredentialStore.java

```diff
@@ -33,17 +33,16 @@
  *
  * <p>This class is designed for storing real-world identity credentials such as
  * Mobile Driving Licenses (mDL) as specified in ISO/IEC 18013-5:2021. It is however
- * not tied to that specific credential shape and is designed to hold any kind of
- * credential, regardless of shape, presentation-, or issuance-protocol used.
+ * not tied to that specific credential format and is designed to hold any kind of
+ * credential, regardless of format, presentation-, or issuance-protocol used.
  *
  * <p>This code relies on a Secure Area for keys and this dependency is abstracted
  * by the {@link SecureArea} interface and allows the use of different {@link SecureArea}
- * implementations for <em>Credential Key</em> and <em>Authentication Keys</em>) used
- * in the credentials stored in the Credential Store.
+ * implementations for <em>Authentication Keys</em>) associated with credentials stored
+ * in the Credential Store.
  *
- * <p>It is guaranteed that calls to {@link #createCredential(String, SecureArea, SecureArea.CreateKeySettings)},
- * {@link #createCredentialWithExistingKey(String, SecureArea, SecureArea.CreateKeySettings, String)},
- * and {@link #lookupCredential(String)} will return the same {@link Credential} instance
+ * <p>It is guaranteed that calls to {@link #createCredential(String)} and
+ * {@link #lookupCredential(String)} will return the same {@link Credential} instance
```

Comment:
The wording makes it seem like `createCredential(String)` with a credential name which already existing does _not_ override the existing credential, which isn't true given the `testCaching()` test in `CredentialStoreTest`. 
```suggestion
 * <p>It is guaranteed that once a credential is created with 
 * {@link #createCredential(String)}, each subsequent call to 
 * {@link #lookupCredential(String)} will return the same {@link Credential} instance
```

Reply:
OK, makes sense. Accepted, thanks!